android/src/main/kotlin/com/mparticle/mparticle_flutter_sdk/MparticleFlutterSdkPlugin.kt

```diff
@@ -119,6 +119,16 @@ class MparticleFlutterSdkPlugin: FlutterPlugin, MethodCallHandler {
         } ?: result.error(TAG, "Missing attributeKey", null)
       }
       "aliasUsers" -> this.aliasUsers(call, result)
+      "getAttributions" ->
```

Comment:
I'm comparing this to the [java code](https://github.com/mParticle/react-native-mparticle/blob/master/android/src/main/java/com/mparticle/react/MParticleModule.java#L373) over in the react native implementation. It looks like there's a `getLink` being called there. Is that missing here?

Reply:
Oh yea, that is a good callout. I was following along on the iOS implementation and I guess that is an Android-only field?

I'll add it in rn

Reply:
there's also a null check from react side, do we only add it if it's not null?

Reply:
True.  Should there be a check for `entry.value` for each put?

Reply:
it _shouldn't_ ever be null, but it's not enforced so I'm going to add it just to be safe